scrapy/spiders/crawl.py

```diff
@@ -95,9 +97,15 @@ class CrawlSpider(Spider):
     def __init__(self, *a: Any, **kw: Any):
         super().__init__(*a, **kw)
         self._compile_rules()
+        if method_is_overridden(self.__class__, CrawlSpider, "_parse_response"):
+            warnings.warn(
+                "CrawlSpider._parse_response method is deprecated: "
+                "it will be removed in future Scrapy releases. "
+                "Please override CrawlSpider.parse_with_rules method instead."
```

Comment:
```suggestion
                "The CrawlSpider._parse_response method is deprecated: "
                "it will be removed in future Scrapy releases. "
                "Please override the CrawlSpider.parse_with_rules method instead."
```